tests/cases/fourslash/codeFixInferFromUsageExistingJSDoc.ts

```diff
@@ -0,0 +1,90 @@
+/// <reference path='fourslash.ts' />
+// @allowJs: true
+// @checkJs: true
+// @noImplicitAny: true
+// @strictNullChecks: false
+// @Filename: important.js
+
+/////** @param x no types here! */
+/////**
+//// * 1
+//// * @param x a duplicate!
+//// * @param y yy
+//// */
+/////**
+//// * 2
+//// * @param z zz
+//// */
+////function f(x) {
+////    return x * 1
+////}
+////
+////var o = {
+////    /** 1
+////     * @return First one
+////     */
+////    /** 2
+////     * @see also
+////     */
+////    /** 3
+////     * @return Second one
+////     * @extends {C<number>} nothing really
+////     * @class
+////     * @this {*} doesn't make sense here
+////     * @enum wat
+////     */
+////    /**
+////     * @typedef {number} Meter or something
+////     * @typedef {Object} Position Comment!
+////     * @property {number} x what a horrible place for a type
+////     * @property {number} y please don't do this
+////     */
+////    /**
+////     * @template {string} T postfix comment
+////     * @callback Action not sure what this will do
+////     * @param {T} thing
+////     * @returns {string} oh no
+////     */
+////    get m() { return undefined }
+////}
+////o.m = 1
+
+verify.codeFixAll({
+    fixId: "inferFromUsage",
+    fixAllDescription: "Infer all types from usage",
+    newFileContent:
+`/**
+ * 1 2
+ * @param {number} x no types here!
+ * @param x a duplicate!
+ * @param y yy
+ * @param z zz
+ */
+function f(x) {
+    return x * 1
+}
+
+var o = {
+    /**
+     * 1 2 3
```

Comment:
Should these not be newline-delineated, since they did originally come from different lines, after all?

Reply:
Yeah, you are right. I forgot that real life comments are not "1" or "2", but complete sentences along the lines of "Crunches and munches input to produce small edible bricks."

Reply:
https://www.thisiswhyimbroke.com/solid-chocolate-lego-bricks/

Reply:
I've only ever had this kind: https://www.ohnuts.com/buy.cfm/bulk-candy/unwrapped-candy/blox